### PR TITLE
test: ERC1155 set approval for all e2e

### PR DESCRIPTION
**Pull request by C-Ivan**
## **Description**

This PR is within the scope of the Quality Quest.

We're including an e2e test case to check the set approval for all for ERC1155 in test dApp.

**Test steps:**
Given I am on the test dapp
When I tap on the Set Approval for All button under the ERC 1155 section
Then the transaction bottom sheet should appear
When I submit the transaction
Then the transaction should appear in the transaction history

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [✓] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [✓] I've completed the PR template to the best of my ability
- [✓] I’ve included tests if applicable
- [✓] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [✓] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/pages/Browser/TestDApp.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ class TestDApp {
   get nftSetApprovalForAllButton() {
     return Matchers.getElementByWebID(
       BrowserViewSelectorsIDs.BROWSER_WEBVIEW_ID,
-      TestDappSelectorsWebIDs.SET_APPROVAL_FOR_ALL_BUTTON_ID,
+      TestDappSelectorsWebIDs.SET_APPROVAL_FOR_ALL_NFT_BUTTON_ID,
```
Comment (C-Ivan) [inline e2e/pages/Browser/TestDApp.js:92]: Renaming for better readability

File: e2e/pages/Browser/TestDApp.js [hunk 2]
```diff
@@ -100,6 +100,13 @@ class TestDApp {
     );
   }
 
+  get erc1155SetApprovalForAllButton() {
```
Comment (C-Ivan) [inline e2e/pages/Browser/TestDApp.js:103]: Selector object for ERC1155 set approval for all button

File: e2e/pages/Browser/TestDApp.js [hunk 3]
```diff
@@ -146,6 +153,10 @@ class TestDApp {
     await this.tapButton(this.nftSetApprovalForAllButton);
   }
 
+  async tapERC1155SetApprovalForAllButton() {
```
Comment (C-Ivan) [inline e2e/pages/Browser/TestDApp.js:156]: Tapping the ERC1155 set approval for all button

File: e2e/specs/confirmations/set-approval-for-all-erc1155.spec.js
Comment (C-Ivan) [inline e2e/specs/confirmations/set-approval-for-all-erc1155.spec.js:1]: New set approval for all ERC1155 e2e test case

**Comment by EtherWizard33** [comment]
LGTM, but I'll let Curtis confirm all is good

**Review by cortisiko — APPROVED** [review]

**Pull request merged**